.ci/scripts/rtc_s1/test_int_rtc_s1.sh

```diff
@@ -90,7 +84,7 @@ fi
 echo "Creating scratch directory $scratch_dir."
 mkdir -p --mode=777 "$scratch_dir"
 
-
+expected_dir="${TMP_DIR}/${EXPECTED_DATA%.*}/"
```

Comment:
Hey Scott, in cslc_s1 weren't you able to simplify this statment, usign ${EXPECTED_DIR} or $expected_dir (sorry i forgot the exact change).  Not use if it's worth changing for consistency?

Reply:
It needed to be changed in the comparison script that runs within the container. For the integration test script this is the correct statement.